ledger/metrics/src/main/scala/com/daml/telemetry/TelemetryContext.scala

```diff
@@ -146,7 +127,7 @@ protected class DefaultTelemetryContext(protected val span: Span) extends Teleme
   protected def createSubSpan(
       spanName: String,
       kind: SpanKind,
-      attributes: (AttributeKey[String], String)*
+      attributes: (SpanAttribute, String)*
   ): Span = {
     val subSpan =
       ParticipantTracer
```

Comment:
@miklos-da do you know what would be the consequences of using a global `tracer` across the sdk and the oem-integration-kit? I'm wondering if I really have to make it a parameter.

Reply:
According to my e2e test, there were correct service names in the UI.

Reply:
That's fine, based on the [OpenTelemetry documentation](https://github.com/open-telemetry/opentelemetry-java/blob/005b8c97efbc1d20ad1565922a1176d503f439af/api/all/src/main/java/io/opentelemetry/api/GlobalOpenTelemetry.java#L105) this only means fixing "the name of the instrumentation library, not the name of the instrument*ed* library". We can consider `com.daml.telemetry` as the instrumentation library.

Reply:
Thanks, I changed the instrumentation name from `participant` to `com.daml.telemetry` as discussed offline.